### DISCUSSION TITLE
Having this  SSL connection timeout while trying to install NVM on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by unyimix** [coreybutler/nvm-windows#1025] [Support]
![image](https://github.com/coreybutler/nvm-windows/assets/66772195/bf792688-c659-408c-b7b9-c889daf4494f)
![image](https://github.com/coreybutler/nvm-windows/assets/66772195/7f48a262-5430-424d-8489-9aee5cfcf78f)

**Comment by coreybutler** [comment]
**NVM _for Windows_** doesn't run on Ubuntu. I think you're looking for [nvm-sh](https://github.com/nvm-sh/nvm).
